### PR TITLE
Add 332 Reconstruct Itinerary

### PR DESCRIPTION
**Pull request by xliu45**

### CODE REVIEW

**Comment by xliu45** [comment]
@sasa150 @lvyb @FreeTymeKiyan @JoannaCode @kuangami

File: C++/332_Reconstruct_Itinerary.cpp [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+        for (int i = 0; i < tickets.size(); i++){
+            m[tickets.at(i).first][tickets.at(i).second]++;
+        }
+        string from = "JFK";
+        vector <string> itinerary;
+        itinerary.push_back(from);
+        findpath(from, itinerary, tickets.size()+1 , m);
+        return (itinerary);
+    }
+    bool findpath(string from, vector<string>& itinerary, int sz, unordered_map<string, map<string, int>> &m){
```
Comment (forest5013) [inline C++/332_Reconstruct_Itinerary.cpp]: you define this function as bool but you never use the return value in your function

Comment (sasa150) [inline C++/332_Reconstruct_Itinerary.cpp]: interesting problem, mark.

**Comment by xliu45** [comment]
@forest5013  I did use it in
if (findpath(index->first, itinerary, sz , m)) return true;

**Pull request merged**